### PR TITLE
Feature/emails fixes

### PR DESCRIPTION
**Pull request by iancj97**
Emails work across multiple floor!

### CODE REVIEW

**Comment by iancj97** [comment]
Please God, let's have this merged in before we present.

**Comment by iancj97** [comment]
Is grammatically correct, sends cropped emails and all.

File: src/Map/Boundary/MapBoundary.java [hunk 1]
```diff
@@ -100,12 +126,11 @@ public Hospital getHospital()
     private int incrementFloor(int incAmount)
     {
         int nextFloorID = currentFloor.getFloorNumber() + incAmount;
-
         if(kioskFloor==null)
         {
             System.out.println("You have not specififed a kiosk");
         }
-        else if(nextFloorID<=kioskFloor.getBuilding().getFloors().size() &&( nextFloorID >0))
```
Comment (benhylak) [inline src/Map/Boundary/MapBoundary.java]: Does building switching work or not?

File: src/Map/Boundary/MapBoundary.java [hunk 2]
```diff
@@ -115,7 +140,7 @@ else if(nextFloorID<=kioskFloor.getBuilding().getFloors().size() &&( nextFloorID
                 }
                 else
                 {
-                    changeFloor(kioskFloor.getBuilding().getFloor(nextFloorID));
+                    changeFloor(currentBuilding.getFloor(nextFloorID));
```
Comment (benhylak) [inline src/Map/Boundary/MapBoundary.java]: IF not, please revert your attempt so I can start from scratch

**Comment by iancj97** [comment]
Let's merge this in and make a new branch, so we know that develop has functional directions.

**Comment by benhylak** [comment]
src/Application/Database/CustomFilePath.java
src/Map/Boundary/MapBoundary.java
src/Map/Controller/MapEditorController.java
src/Map/Controller/UserDirectionsPanel.java
src/Map/Entity/Floor.java
src/Map/Entity/ProxyImage.java
src/Map/Navigation/DirectionFloorStep.java
src/Map/Navigation/Guidance.java
src/Map/Navigation/SendEmail.java
src/Map/Navigation/TextDirectionsCreator.java
src/View/Admin/MapBuilder/MapEditorView.fxml

Which files do I need to merge? Also can you please address my PR comments?

**Comment by iancj97** [comment]
Pull request moved to different branch

**Pull request closed without merging**